sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -115,7 +115,9 @@
    <version ClassVersion="10" checksum="3777054621"/>
   </class>
 
-  <class name="caf::SRTrack" ClassVersion="15">
+  <class name="caf::SRTrack" ClassVersion="17">
+   <version ClassVersion="17" checksum="4075719476"/>
+   <version ClassVersion="16" checksum="2837285941"/>
```

Comment:
Technically we only need to keep the most recent checksum (4075719476) with version 16. The intermediate version never actually made it into a release so doesn't need compatibility.